app/assets/javascripts/lifeitup_layout.js

```diff
@@ -61,7 +61,10 @@ function layout_resizer () {
 // init popovers
 
 $(function () {
-  $('[data-toggle="popover"]').popover()
+  $('body').popover({
+    selector: '[data-toggle="popover"]',
+    trigger: 'focus'

```

Comment:
Do we wanna change this here in lifeitup file? if we update this later on we might forget we changed that.